README.md

```diff
@@ -43,7 +43,12 @@ pip install -e .
 ```
 
 instead. It is recommended to create an virtual environment prior
-to installing this library.
+to installing this library. If you only intend to use this library from the CLI,
+use [pipx](https://pypa.github.io/pipx/) for the installation instead:
+
+```bash
+pipx install anonfile
```

Comment:
This is fine, either way wouldn't have mattered to me. Personally, I like the straight forward pip installation. Very simple. However, I can see the benefits of having this option. We will go with this change, but it's very low priority for me.

Reply:
`pipx` is authored by the Python Packaging Authority and the go-to tool for installing Python applications with global scope on any platform. Using `pip` for global installations is very much discouraged because it can degrade your system quickly, especially on Unix platforms.

Lots of beginners don't know about `pipx` or `venv`, so I think it doesn't hurt to promote best practices for safely getting `anonfiles` up and running on their system.